powsybl-build-tools/src/main/resources/powsybl-build-tools/lombok.config

```diff
@@ -0,0 +1,14 @@
+# Don't search another lombok.config file in parent folders
+config.stopbubbling = true
+
+# We add annotations flags for IDEs and tools :
+#  * mark the code as generated for linters and coverage tools
+lombok.addJavaxGeneratedAnnotation = true
+lombok.addLombokGeneratedAnnotation = true
+#  * disable all warnings on generated code for checking tools and IDE (eg. checkstyle)
+lombok.addSuppressWarnings = true
```

Comment:
Remove ? From the docs
"Lombok adds the @SuppressWarnings("all") annotation to all generated nodes by default."

Reply:
hum, yes, i remove it